Functions.Templates/Templates/MySqlTrigger-JavaScript/function.json

```
@@ -0,0 +1,12 @@
+{
+  "bindings": [
+    {
+      "name": "changes",
+      "type": "mysqlTrigger",
+      "direction": "in",
+      "tableName": "table1",
```

Do you need to add this dummy value, i think empty value will do fine as that will immediately throw the error.

---

Keeping the same pattern used for the previous Mysql templates to keep consistency.